uber/site_sections/preregistration.py

```diff
@@ -191,11 +191,28 @@ def banned(self, id):
         attendee, group = self._get_unsaved(id)
         return {'attendee': attendee}
 
+    def process_free_prereg(self, session):
+        charge = Charge(listify(self.unpaid_preregs.values()))
+        if charge.total_cost <= 0:
+            for attendee in charge.attendees:
+                attendee.paid = c.NEED_NOT_PAY
```

Comment:
This should be removed -- we set their paid status in a presave adjustment, and even if we wanted to explicitly put it here, you only did it for attendees who are not in groups.